rose/server/score_test.py

```diff
@@ -227,14 +227,14 @@ def test_forward(self):
         self.obstacle = obstacles.PENGUIN
         self.track.set(self.x, self.y, self.obstacle)
         self.process()
-        # Player keep position but get more life
+        # Player keep position but get more score
         assert self.player.x == self.x
         assert self.player.y == self.y
-        assert self.player.life == self.life + 1
+        assert self.player.score == self.score + 1
```

Comment:
should use config, not hardcoded value. Otherwise we are testing the config instead of the logic.